client/src/components/SignCampaign.js

```diff
@@ -48,22 +51,39 @@ class SignCampaign extends Component {
   };
 
   checkSignIn = () => {
+    function FieldGroup({ id, label, ...props }) {
+      return (
+        <FormGroup controlId={id}>
+          <ControlLabel>{label}</ControlLabel>
+          <FormControl {...props} />
+        </FormGroup>
+      );
+    }
     if (this.props.auth && (!this.props.auth.googleID && !this.props.auth.facebookID)) {
       return (
-        <div>
-          <a className="google-button-signature" href="/auth/google">
-            <GoogleButton label="Google" />
-          </a>
-          <a className="facebook-button-signature" href="/auth/facebook">
-            Sign In With Facebook!
-          </a>
-        </div>
+        <Row>
+          <Col md={12}>
+            <a className="google-button-signature" href="/auth/google">
+              <GoogleButton label="Google" />
+            </a>
+            <h5 className="content text-center">OR</h5>
+            <a className="google-button-signature" href="/auth/google">
```

Comment:
Should `<GoogleButton />` be repeated here?

Reply:
No! Need to either make custom custom Facebook button or pull in a new package.